ci/travis/36-minimal.yaml

```diff
@@ -14,6 +14,9 @@ dependencies:
   - pytest
   - pytest-cov
   - codecov
+  - fsspec
+  - requests
+  - aiohttp
```

Comment:
Why are requests and aiohttp needed? (didn't see any tests or so that use them?)

Reply:
They are needed for the specific `fsspec` backend that I used in the tests (but not the top-level package), i.e. [here](https://github.com/intake/filesystem_spec/blob/88f6cb9004b5fa3fa2883fa8f01116e4ca7e77ac/fsspec/registry.py#L100).

Reply:
That's only for the test with fsspec reading a remote file, right, while we already have two other tests using local files? 

Do you think it adds value to have the http test as well, assuming that in both cases `with fsspec.open(..., "r") as f:` will result in a similar file-like object? 
(in which case I would leave out that test, and otherwise please add a comment in the yaml files indicating why those packages are included)

Reply:
Sure, I suppose it doesn't add a ton of value to have the remote file test, so long as the file-like object can be trusted to produce bytes. I can remove that test.